docs/source/intro.rst

```diff
@@ -4,6 +4,7 @@ Introduction
 The ``eXtension interface``, also called ``CORE-V-XIF``, is an interface aimed at extending a |processor| with (custom or standardized) instructions implemented in a |coprocessor|.
 
 It can be used to implement standard RISC-V extensions as for example B (Bit Manipulation), M (Integer Multiplication and Division), F (Single-Precision Floating Point) and D (Double-Precision Floating Point). It can also be used to implement custom extensions.
+Extensions implemented on the interface are unprivileged extension, i.e. implementing privileged extensions like H (Hypervisor) is not supported.
```

Comment:
unprivileged extension -> unprivileged extensions

Reply:
fixed in 9f6b67d